backend/logic/courses/courses.js

```diff
@@ -145,9 +147,26 @@ var addCourseOfferings = async courseOfferings => {
   }
 };
 
+let phaseOutCourse = async courseCode => {
```

Comment:
Make sure you do a validation check for the course code.

Reply:
true, thanks!

Reply:
Try a regex expression, this is just a quick one but it should match 4 capital letters a single space and 3 digits 0-9
use https://regex101.com/ to help you if you do regex
```regex
courseCode.match(/[A-Z]{4}[ ]{1}\d{3}/g)
```

Reply:
also, i think someone made a function for that (at least i saw it on @GarethDPeters's branch, so you could just reuse that.

Reply:
done! thanks :D

Reply:
done!